### PR TITLE
Added ability to increase or decrease font size

### PR DESCRIPTION
**Pull request by thewhitetulip**
Two new commands added, IncreaseFontSize and DecreaseFontSize. The unit test case passes.

### CODE REVIEW

File: font_test.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	ed := backend.GetEditor()
+	w := ed.NewWindow()
+	defer w.Close()
+	// Set the default font size to 12
+	// Run the command
+	// check the font size
+	// if increased by 2 then ok, else fail.
+	w.Settings().Set("font_size", 12)
+
+	previousFontSize := w.Settings().Get("font_size").(int)
```
Comment (ricochet1k) [inline font_test.go]: Use .Int("font_size")

File: font_test.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+)
+
+func TestIncreaseFontSize(t *testing.T) {
+	ed := backend.GetEditor()
+	w := ed.NewWindow()
+	defer w.Close()
+	// Set the default font size to 12
+	// Run the command
+	// check the font size
+	// if increased by 2 then ok, else fail.
```
Comment (ricochet1k) [inline font_test.go]: These comments aren't necessary anymore

File: font.go [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+	// DecreaseFontSize command decreases the font size.
+	DecreaseFontSize struct {
+		backend.DefaultCommand
+	}
+)
+
+// Run executes the IncreaseFontSize command.
+func (i *IncreaseFontSize) Run(w *backend.Window) error {
+	fontSize := w.Settings().Get("font_size").(int)
+	fmt.Println("Font size found was " + string(fontSize))
```
Comment (ricochet1k) [inline font.go]: This Println shouldn't be here anymore

File: font_test.go (hunk 1 above)
Comment (thewhitetulip) [inline font_test.go]: Out of curiosity, what is the difference between .Int() and .Get().(int)?

File: font.go [hunk 4]
```diff
@@ -0,0 +1,47 @@
+
```
Comment (ricochet1k) [inline font.go]: Remove this blank line to fix Travis.

File: font_test.go (hunk 2 above)
Comment (thewhitetulip) [inline font_test.go]: I thought comments would be great.

File: font.go (hunk 3 above)
Comment (thewhitetulip) [inline font.go]: Yes, you are right.

File: font.go (hunk 4 above)
Comment (thewhitetulip) [inline font.go]: Which blank line? Every other module has a blank line before package and after the license text.

File: font.go [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+	}
+	// DecreaseFontSize command decreases the font size.
+	DecreaseFontSize struct {
+		backend.DefaultCommand
+	}
+)
+
+// Run executes the IncreaseFontSize command.
+func (i *IncreaseFontSize) Run(w *backend.Window) error {
+	fontSize := w.Settings().Get("font_size").(int)
```
Comment (ricochet1k) [inline font.go]: Use .Int("font_size")

File: font_test.go [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+	"github.com/limetext/backend"
+)
+
+func TestIncreaseFontSize(t *testing.T) {
+	ed := backend.GetEditor()
+	w := ed.NewWindow()
+	defer w.Close()
+	w.Settings().Set("font_size", 12)
+
+	previousFontSize := w.Settings().Get("font_size").(int)
```
Comment (ricochet1k) [inline font_test.go]: Use .Int("font_size")

File: font_test.go [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+func TestDecrementFontSize(t *testing.T) {
+	ed := backend.GetEditor()
+	w := ed.NewWindow()
+
+	defer w.Close()
+	w.Settings().Set("font_size", 12)
+
+	previousFontSize := w.Settings().Get("font_size").(int)
```
Comment (ricochet1k) [inline font_test.go]: use .Int("font_size")

File: font.go [hunk 8]
```diff
@@ -0,0 +1,41 @@
+// Copyright 2014 The lime Authors.
```
Comment (erbridge) [inline font.go]: The year is 2016.

File: font_test.go [hunk 9]
```diff
@@ -0,0 +1,37 @@
+// Copyright 2013 The lime Authors.
```
Comment (erbridge) [inline font_test.go]: The year is 2016.

File: font.go [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+	// DecreaseFontSize command decreases the font size.
+	DecreaseFontSize struct {
+		backend.DefaultCommand
+	}
+)
+
+// Run executes the IncreaseFontSize command.
+func (i *IncreaseFontSize) Run(w *backend.Window) error {
+	fontSize := w.Settings().Int("font_size")
+	fontSize = += 2
```
Comment (erbridge) [inline font.go]: `= +=` seems like an error to me.

File: font.go [hunk 11]
```diff
@@ -0,0 +1,41 @@
+// Copyright 2014 The lime Authors.
+// Use of this source code is governed by a 2-clause
+// BSD-style license that can be found in the LICENSE file.
+
+package commands
+
+import "github.com/limetext/backend"
+
+type (
+	// IncreaseFontSize command increases the font size.
```
Comment (erbridge) [inline font.go]: Document that the font sizes change by 2 units at a time. Also, is that what ST3 does?

File: font.go (hunk 10 above)
Comment (thewhitetulip) [inline font.go]: I was running tests in another branch, so this went undetected :D

File: font.go (hunk 11 above)
Comment (thewhitetulip) [inline font.go]: Documented, I do not know exactly, but when I used the command on ST3, it did look like a change in 2 units at a time.

File: font.go (hunk 10 above)
Comment (erbridge) [inline font.go]: Make sure you're developing on the same branch you're testing and pushing.

File: font.go (hunk 11 above)
Comment (erbridge) [inline font.go]: Just checked, and on my system at least, it's a change of 1 per call.

File: font.go [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+	// DecreaseFontSize command decreases the font size by 2.
+	DecreaseFontSize struct {
+		backend.DefaultCommand
+	}
+)
+
+// Run executes the IncreaseFontSize command.
+func (i *IncreaseFontSize) Run(w *backend.Window) error {
+	fontSize := w.Settings().Int("font_size")
+	fontSize += 2
```
Comment (erbridge) [inline font.go]: ICYMI,

> Just checked, and on my system at least, it's a change of 1 per call.

Comment (thewhitetulip) [inline font.go]: Okay.

**Comment by erbridge** [comment]
LGTM. Thanks :).

**Comment by thewhitetulip** [comment]
@erbridge My pleasure! I want lime to be usable as soon as possible, I am not happy with VSCode.

**Pull request merged**